models/screenshot_service.go

```diff
@@ -36,3 +39,22 @@ func (s *ScreenshotService) FindAll(appVersion *AppVersion) ([]Screenshot, error
 	}
 	return screenshots, nil
 }
+
+// BatchUpdate ...
+func (s *ScreenshotService) BatchUpdate(screenshots []Screenshot, whitelist []string) ([]error, error) {
```

Comment:
It's a bit confusing that the method returns multiple error types, how about naming the returns?
```suggestion
func (s *ScreenshotService) BatchUpdate(screenshots []Screenshot, whitelist []string) (validationErrors []error, sqlError error) {
```